src/index.html

```diff
@@ -1,13 +1,339 @@
+              <h3 class="detail__description-title">
+                Range & Integrated lights
+              </h3>
+              <p class="detail__description-text">
+                The removable battery has up to 70km battery autonomy and weighs
+                only 2.4 kg. Lights integrated into the frame give you always-on
+                visibility day and night.
+              </p>
+            </div>
+          </div>
+
+          <div class="detail gallery__detail">
+            <div class="detail__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--desktop-1-4">
+                  <div class="detail__photo-container">
+                    <img
+                      src="images/details/5.jpg"
+                      alt="Auto Unlock"
+                      class="detail__photo"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item--desktop-5-12">
+                  <div class="detail__photo-container">
+                    <img
+                      src="images/details/6.jpg"
+                      alt="Auto Unlock"
+                      class="detail__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="detail__description">
+              <h3 class="detail__description-title">
+                Hydraulic disc brakes & Lightweight
+              </h3>
+              <p class="detail__description-text">
+                Brakes with total stopping power the second you make contact.
+                The removable battery has up to 70km battery autonomy and weighs
+                only 2.4 kg. Lights integrated into the frame give you always-on
+                visibility day and night.
+              </p>
+            </div>
+          </div>
+
+          <div class="grid grid__desktop">
+            <div class="grid__item grid__item--5-8">
+              <a href="#" class="button"> Explore </a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">Contact Us</h2>
+          <div class="contact-us__block">
+            <div class="contact-us__inner contact-us__inner--form">
+              <form action="get" id="form">
+                <input
+                  name="name"
+                  type="text"
+                  class="contact-us__field form-field"
+                  placeholder="Name"
+                  required
+                >
+                <input
+                  name="email"
+                  type="email"
+                  class="contact-us__field form-field"
+                  placeholder="Email"
+                  required
+                >
+                <textarea name="message" class="contact-us__field form-field form-field--message"></textarea>
```

Comment:
Here is missing placeholder 'message' and the inputs using Arial font-family.
![image](https://user-images.githubusercontent.com/63786090/209816226-11300a18-ae8b-4840-92d7-babbbe4fb002.png)

Reply:
Yap. here I had strange but. I don`t know why but my placeholder was invisible, so I just cutted this part. I'll change it and pray for its vision